contracts/Core/BlockManager.sol

```diff
@@ -278,10 +277,10 @@ contract BlockManager is ACL, BlockStorage {
     {   
         // generating pseudo random number (range 0..(totalstake - 1)), add (+1) to the result,
         // since prng returns 0 to max-1 and staker start from 1
-        if ((Random.prng(10, stakeManager.getNumStakers(), keccak256(abi.encode(iteration)))+(1)) != stakerId) {
+        if ((Random.prng(10, stakeManager.getNumStakers(), keccak256(abi.encode(iteration)), constants.epochLength())+(1)) != stakerId) {
```

Comment:
Why was this done?

Reply:
Its because Random.prng() method is changed.
It didn't have epoch length as function param before
Why @dev1644 added it I think
Is without this passing, we would have to inherit constants.sol in random as well, because prng uses it 

Old
https://github.com/razor-network/contracts/blob/8f59a1e78f886bedb433c1fcd3592af1296753f6/contracts/lib/Random.sol#L26

Reply:
Correct @abhishekvispute , thus reducing the size of  `Random Library` & minimizing any redundant call.